### DISCUSSION TITLE
How to use in EXPO GO

### DISCUSSION THREAD

**Discussion by iman2420** [rsuite/rsuite#2015] [Q&A]
I want to use this project in expo to run in Android, IOS and web. so how to implement this awesome library in expo?

**Comment by SevenOutman** [comment]
I don't have any experience in expo. @simonguo Any idea?

**Comment by simonguo** [comment]
Hi @iman2420  
The interactive design of rsuite is mainly for desktop web applications and is not recommended for use on mobile devices.
Perhaps Material-UI may be more suitable for mobile applications. (https://docs.expo.dev/guides/userinterface/)

**Comment by suhaotian** [comment]
this is only for web. maybe you can use it in expo web, but not android/ios
